cpp/test/core/maybe_tests.cpp

```diff
@@ -1,28 +1,41 @@
 #include "precompiled.h"
 #include "maybe_tests.h"
 
+#include "allocators.h"
+
+#include <boost/none.hpp>
+#include <boost/optional/optional.hpp>
+#include <boost/static_assert.hpp>
+
+#include <initializer_list>
+#include <iterator>
+#include <map>
+#include <vector>
+
 void ConstInterface(const bond::maybe<int8_t>& x)
 {
-    bond::maybe<int8_t> z;
-    int8_t y;
+    BOOST_TEST_CONTEXT("Const interface")
```

Comment:
Looks like VC12 is still using boost 1.58 which does not have `BOOST_TEST_CONTEXT`.

Reply:
Sigh. I have a fix for that to productionalize...